rwatch/ui/layer/action_bar_layer.c

```diff
@@ -116,7 +117,7 @@ static void draw(Layer *layer, GContext *context)
             // Draw the icon:
             GSize icon_size = action_bar->icons[i]->raw_bitmap_size;
 #ifdef PBL_RECT
-            graphics_draw_bitmap_in_rect(context, action_bar->icons[i], GRect(full_bounds.size.w - icon_size.w - 2, y - (increment / 2) - (icon_size.h / 2), icon_size.w, icon_size.h));
+            graphics_draw_bitmap_in_rect(context, (struct GBitmap *)action_bar->icons[i], GRect(full_bounds.size.w - icon_size.w - 2, y - (increment / 2) - (icon_size.h / 2), icon_size.w, icon_size.h));
```

Comment:
why the cast here? this will remove the `const` modifier, which shouldn't be necessary